central/jwt/jwt.go

```diff
@@ -63,7 +96,16 @@ func create() (tokens.IssuerFactory, tokens.Validator, error) {
 		return nil, nil, errors.Wrap(err, "parsing private key")
 	}
 
-	return tokens.CreateIssuerFactoryAndValidator(issuerID, privateKey, keyID)
+	factory, validator, err := tokens.CreateIssuerFactoryAndValidator(issuerID, privateKey, keyID)
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "creating factory and validator")
+	}
+	validator = &m2mValidator{
```

Comment:
Won't demand a change here but I definitely prefer having a new var name for a new object, makes reading easier for no extra cost.

Reply:
Added a comment, and removed the unnecessary variable override.